simple_app/templates/index_simple.html

```diff
@@ -28,16 +28,51 @@ <h1 class="display-2">Welcome!</h1>
         astrometry, photometry, spectra, and image data for <strong>{{ source_count|safe }}</strong>
         sources from the literature.<br>
         You can search the database through either an <a href="{{ url_for('search') }}">object search</a>,
-        a <a href="{{ url_for('fulltextsearch') }}">free text search</a> or a
-        <a href="{{ url_for('coordquery') }}">coordinate query</a>.
+        a <a href="{{ url_for('full_text_search') }}">free text search</a> or a
+        <a href="{{ url_for('coordinate_query') }}">coordinate query</a>.
         You can also view the positions and photometry of the entire database via our
-        <a href="{{ url_for('load_multiplot') }}">interactive viewer</a>!<br>
+        <a href="{{ url_for('load_multi_plot') }}">interactive viewer</a>!<br>
         Navigation can be performed through the use of the header bar, whilst details of our GitHub respositories are
         available through the footer.
         Any clickable table titles indicate that that data can be downloaded as either a .csv or .zip file.
         We welcome contributions from the entire community and greatly appreciate any feedback (including bug reports)!
     </p>
 </div>
+<div class="container-fluid">
+    <h2 class="display-3">Figures</h2>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="p-2 col-xxl-6 float-start">
```

Comment:
Something is off about the alignment with this div class. The figures on my end are half aligned to the left, half to the right and a huge space between them:
![Screen Shot 2023-07-14 at 5 43 06 PM](https://github.com/SIMPLE-AstroDB/SIMPLE-web/assets/13004332/998813f2-5edb-48db-b022-ec68f75c72fd)

Reply:
I've swapped it now to be 3 figures, so some black space only appears on full screen 4k. I don't want the images stretching to fill space as they look nice with the aspect ratio as is.